### DISCUSSION TITLE
Welcome to ganja.js Discussions!

### DISCUSSION THREAD

**Discussion by enkimute** [enkimute/ganja.js#97] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kungfooman** [comment]
I've seen this Welcome post template before 😅 

I am interested in learning more about math in general and Ganja seems like a great/dynamic/fun way to do so. I hope I can contribute whatever I can and to see this project florish.

I deeply respect the amount of work you put into this and I am looking forward to see where this will continue to develop in the future.

My background is rather "unmathy" besides some game programming (stuff like C and Windows/Linux programming). Yet I am happy to experiment around and learn more about this.

**Comment by pedroth** [comment]
Great project @enkimute !! Will the project be adding more languages to the code generation?  Kotlin or Java? Thanks
